src/components/breadcrumb/FilterAndSelected.tsx

```diff
@@ -1,7 +1,15 @@
 import React, { useMemo } from 'react';
+import { I18nextManager } from 'tdp_core';
 import { getAllFilters } from '../../store/storeUtils';
 import { useAppSelector } from '../../hooks/useAppSelector';
 
+function equalArrays<T>(a: T[], b: T[]) {
+  if (a.length !== b.length) {
+    return false;
+  }
+  return a.every((ai, i) => ai === b[i]);
+}
+
```

Comment:
would the lodash isEqual function work here? just to avoid a custom way of doing it, and we use lodash functions in other places